.github/actions/build-zui/action.yml

```diff
@@ -69,7 +69,7 @@ runs:
       env:
         GH_TOKEN: ${{ inputs.gh_token }}
         APPLE_ID: ${{ inputs.apple_id }}
-        APPLE_ID_PASSWORD: ${{ inputs.apple_id_password }}
+        APPLE_APP_SPECIFIC_PASSWORD: ${{ inputs.apple_id_password }}
```

Comment:
As we transition from the separate `electron-builder-notarize` to leveraging the notarization support that's in `electron-builder`, the name of this environment variable changes but everything else stays the same.